### PR TITLE
Implementation of Slideshows in 2.0

### PR DESCRIPTION
**Pull request by petervnguyen99**
Implementation of Slideshows in dcf_starter for code written in dcf.

In order to test the slideshow I had to add `dcf-slideshow.js` and `dcf-slideshow-theme.js` into `dcf_starter/node_modules/dcf/js`.

I spent a day trying to implement it only to realize that I didn't have the most updated version of 2.0 so I deleted the old branch, updated 2.0, and created this one.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,6 +17,104 @@
     Header markup here
   </header>
   <main class="dcf-wrapper" id="dcf-main" tabindex="-1" role="main">
+    <!-- Slideshow WIP -->
+    <style>
+      .dcf-slideshow ul {
```
Comment (jsturek) [inline index.html:22]: Should we put these into dcf scss now or do we plan to do it in phases?  It might be a good learning opportunity for Peter to do it now knowing we will likely tweak it later.

File: index.html [hunk 2]
```diff
@@ -17,6 +17,104 @@
...
+      /* Fallback styles */
+      .dcf-slideshow ul:first-child li {
+        flex: 0 0 84%; /* Make sure some of the next slide appears */
+      }
+      /* Styles after JavaScript loads */
+      .dcf-slideshow ul:first-child li.dcf-slide {
+        flex: 0 0 100%; /* Make each slide fill the full width of the slideshow */
+      }
+
+      .unl .dcf-slide-caption[aria-hidden="true"] {
```
Comment (jsturek) [inline index.html]: Base DCF does not know about `unl` class should change to `theme`

File: index.html [hunk 3]
```diff
@@ -135,7 +315,120 @@ <h2>Example Modal 2</h2>
 
     // JavaScript Support Example
     DCFUtility.flagSupportsJavaScript();
-    </script>
+
+    const keyframesClose1 = [
```
Comment (jsturek) [inline index.html:319]: Why is this here?  Isn't this in DCF `dcf-slideshow-theme.js`

**Review by jsturek — CHANGES_REQUESTED** [review]
See comments

File: index.html (hunk 3 above)
Comment (petervnguyen99) [reply index.html:319]: Ryan says that dcf-slideshow-theme is something that a developer will customize for his/her own theme so it should be in dcf_starter and eventually wdn.

Comment (jsturek) [reply index.html:319]: ok

File: index.html (hunk 1 above)
Comment (jsturek) [reply index.html:22]: @skoolbus39 ^^^ ?

File: index.html (hunk 3 above)
Comment (jsturek) [reply index.html:319]: This is fine but we'll need to make sure the dcf placeholder javascript `dcf-slideshow-theme.js` and sample here in dcf_starter stay in sync or just manage it in dcf_starter some how.  We would need to at least lint check it if only in starter.

File: index.html (hunk 1 above)
Comment (jsturek) [reply index.html:22]: @skoolbus39 - these non-theme CSS items are the only things that I would change. They probably should be part of DCF SCSS and not here id DCF starter.  If you want to do that in another task that's fine since not sure if Peter is available until Fall. I've already tentatively approved the DCF PR for slideshow depending on this issue.
